public/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta name="description" content="Web site created using create-react-app" />
```

Comment:
should we change the site's description here

Reply:
addressed https://github.com/OpenSRP/web/pull/9/commits/a66ca0060ef1e1becf2e39b1893ca5a7161d996a